corehq/apps/export/static/export/js/export_list.js

```diff
@@ -72,30 +72,11 @@ hqDefine("export/js/export_list", [
         assertProperties.assert(pageOptions.urls, ['poll', 'toggleEnabled', 'update']);
 
         var self = ko.mapping.fromJS(options);
-        self.prepareExportError = ko.observable('');
         self.hasEmailedExport = !!options.emailedExport;
-
-        // Unwrap the values in the EMWF filters, turning them into plain {id: ..., text: ...} objects for use with select2
         if (self.hasEmailedExport) {
-            self.emailedExport.filters.emwf_case_filter(_.map(self.emailedExport.filters.emwf_case_filter(), function (mw) {
-                return _.mapObject(mw, function (observable) {
-                    return observable();
-                });
-            }));
-            self.emailedExport.filters.emwf_form_filter(_.map(self.emailedExport.filters.emwf_form_filter(), function (mw) {
-                return _.mapObject(mw, function (observable) {
-                    return observable();
-                });
-            }));
+            self.emailedExport = emailedExportModel(options.emailedExport, pageOptions, self.id(), self.exportType());
```

Comment:
This model (`ExportModel`) had a lot of code associated with updating the saved export, which is represented in data in this `emailedExport` JSON object.  In this commit I moved the associated methods into that object so it's clearer that they don't apply to other export types.